README.md

```diff
@@ -39,7 +40,12 @@ Example-iOS-Apps is an amazing list for people who are beginners and learning iO
 * [swiftui-notes](https://github.com/heckj/swiftui-notes) - A collection of notes, project pieces, playgrounds and ideas on learning and using SwiftUI and Combine.
 * [Harvest-SwiftUI-Gallery](https://github.com/inamiy/Harvest-SwiftUI-Gallery) - Gallery App for Harvest (Elm Architecture + Optics) + SwiftUI + Combine.
 * [SwiftWebUI](https://github.com/SwiftWebUI/SwiftWebUI) - A demo implementation of SwiftUI for the Web
+* [8 Ball](https://github.com/fulldecent/8-ball) - A fully working iOS + watchOS game to make life choices
+
+## iOS Apps written with SceneKit
+* [Pizza Man](https://github.com/fulldecent/pizzaman) - A simple game where you eat flying pepperoni
 
+SceneKit, fully open source, and for sale on App Store
```

Comment:
Please remove this line

Reply:
Fixed in 002f7daae6dfc8a03c8eac8d5ed0a90614902dc4